docs/source/onnxruntime/usage_guides/gpu.mdx

```diff
@@ -0,0 +1,225 @@
+)
+```
+
+[As previously for `CUDAExecutionProvider`](#use-cudaexecutionprovider-with-floatingpoint-model), by passing the session option `log_severity_level = 0` (verbose), we can check in the logs whether all nodes are indeed placed on the TensorRT execution provider or not:
+
+```
+2022-09-22 14:12:48.371513741 [V:onnxruntime:, session_state.cc:1188 VerifyEachNodeIsAssignedToAnEp] All nodes have been placed on [TensorrtExecutionProvider]
+```
+
+### Use TensorRT execution provider with quantized models
+
+When it comes to quantized models, TensorRT supports only models that use **static** quantization with **symmetric quantization** for weights and activations. Thus, to be able to consume with TensorRT models quantized through 🤗 Optimum, the following configuration needs to be passed when doing [static quantization](quantization#static-quantization-example):
+
+```python
+qconfig = QuantizationConfig(
+    ...,
+    is_static=True,
+    activations_symmetric=True,
+    weights_symmetric=True,
+    qdq_dedicated_pair=True,
+    qdq_add_pair_to_weight=True
+)
+```
+
+The parameter `qdq_dedicated_pair=True` is required by TensorRT, that expects a single node after each `QuantizeLinear` + `DequantizeLinear` (QDQ) pair.
+
+The parameter `qdq_add_pair_to_weight=True` is also required by TensorRT, that consumes a graph where the weights are stored in float32 with a QDQ pair. Normally, weights would be stored in fixed point 8-bits format and only a `DequantizeLinear` would be applied on the weights. As such, the storage savings from quantization can not be leveraged when we expect to later use the quantized `.onnx` model with TensorRT.
+
+In the code sample below, after performing static quantization, the resulting model is loaded into the `ORTModel` class using TensorRT as the execution provider. ONNX Runtime graph optimization need to be disabled for the model to be consumed and optimized by TensorRT, and the fact that int8 operations are used needs to be specified to TensorRT.
+
+```python
+session_options = onnxruntime.SessionOptions()
+session_options.graph_optimization_level = onnxruntime.GraphOptimizationLevel.ORT_DISABLE_ALL
+
+model_name = "fxmarty/distilbert-base-uncased-finetuned-sst-2-english-int8-static-symmetric-dedicated-qdq-everywhere"
+ort_model = ORTModelForSequenceClassification.from_pretrained(
+    model_name,
+    from_transformers=False,
+    provider="TensorrtExecutionProvider",
+    session_options=session_options,
+    provider_options={"trt_int8_enable": True},
+)
+```
+
+The model can then be used with the common 🤗 Transformers API for inference and evaluation, such as pipelines:
+```python
+from optimum.pipelines import pipeline
+from transformers import AutoTokenizer
+
+tokenizer = AutoTokenizer.from_pretrained(model_name)
+
+pipe = pipeline(task="text-classification", model=ort_model, tokenizer=tokenizer)
+result = pipe("Both the music and visual were astounding, not to mention the actors performance.")
+print(result)
```

Comment:
Similar comment as above that you can include the actual outputs in the code block